src/main/webapp/WEB-INF/jsp/application/app_form.jsp

```diff
@@ -219,18 +219,27 @@
                                     <spring:message code="absence.period"/>:
                                 </label>
                                 <div class="col-md-9 radio">
-                                    <label class="thirds">
-                                        <form:radiobutton id="fullDay" class="dayLength-full" path="dayLength" checked="checked" value="FULL"/>
+                                    <c:if test="${applicationSettings.allowHalfDays}">
```

Comment:
I would prefer a choose condition for better readability
```
<c:choose>
  <c:when test="${condition1}">
    ...
  </c:when>
  <c:otherwise>
    ...
  </c:otherwise>
</c:choose>
```